package.json

```diff
@@ -21,7 +21,8 @@
     "localenv:build": "docker compose -f ./infrastructure/local/docker-compose.yml -f ./infrastructure/local/peer-docker-compose.yml -f ./infrastructure/local/build-override.yml build",
     "localenv:seed:auth": "pnpm -C ./packages/auth knex seed:run --env=development && pnpm -C ./packages/auth knex seed:run --env=peerdevelopment",
     "localenv:dbvolumes:remove": "docker volume rm local_database-data && docker volume rm local_tigerbeetle-data",
-    "sanity": "pnpm -r build && pnpm -r test"
+    "sanity": "pnpm -r build && pnpm -r test",
+    "docs": "graphql-markdown --update-file ./docs/admin-api.md --heading-level 2 ./packages/backend/src/graphql/schema.graphql"
```

Comment:
Should we either perform this in a pre-commit hook or validate that docs are up to date in 
https://github.com/interledger/rafiki/blob/main/.github/workflows/lint_test_build.yml

Reply:
Yeah, I was wondering about that. I think we should perform it rather than validate it. I'll add it.